spec/plans/add_compiler_spec.rb

```diff
@@ -33,8 +47,9 @@ def allow_standard_non_returning_calls
 
       # {"install_flags"=>
       #   ["--puppet-service-ensure", "stopped",
-      #   "extension_requests:1.3.6.1.4.1.34380.1.3.13=pe_compiler", "extension_requests:1.3.6.1.4.1.34380.1.1.9813=A", "main:certname=compiler"], "server"=>"primary"}
+      #   "extenion_requests:1.3.6.1.4.1.34380.1.3.13=pe_compiler", "extension_requests:1.3.6.1.4.1.34380.1.1.9813=A", "main:certname=compiler"], "server"=>"primary"}
```

Comment:
@ody Is there a typo here? Is this an inadvertent change?

Reply:
Inadvertently deletion of the letter s in extension_requests.